lua/telescope/_extensions/zoxide/list.lua

```diff
@@ -60,20 +60,14 @@ local fuzzy_with_z_score_bias = function(opts)
 end
 
 local entry_maker = function(item)
-  local items = vim.split(string.gsub(item, '^%s*(.-)%s*$', '%1'), " ")
-  local score = 0
-  local item_path = item
-
-  if #items > 1 then
```

Comment:
I can't remember why I added this check when I created this plugin. Is this still handled on your new implementation?

Reply:
I also was curious why this was added - is there cases where you don't produce a score? otherwise it seems that first pre-space object = score, rest = path seems a reliable heuristic.

I fixed this at like 2am and just wanted to provide #11 a fix so I've only been test driving this for 10mins or so so maybe it breaks!